src/demo-app/a11y/card/card-a11y.html

```diff
@@ -0,0 +1,85 @@
+<section>
+  <h2>Husky</h2>
+  <p>Showing a card with title only</p>
+  <md-card class="example-card">
```

Comment:
Should any of these `<md-card>` instances have `role="group"`?
(to line up with the docs https://material.angular.io/components/card/overview)